### PR TITLE
Fix for select Pokemon with less than IV input

### PR DESCRIPTION
**Pull request by stevenzeck**
Fix for #481.

### CODE REVIEW

File: src/me/corriekay/pokegoutil/windows/PokemonTab.java [hunk 1]
```diff
@@ -816,7 +816,8 @@ private void selectLessThanIv() {
         System.out.println("Selecting Pokemon with IV less than: " + ivTransfer.getText());
 
         for (int i = 0; i < pt.getRowCount(); i++) {
-            final double pIv = Double.parseDouble((String) pt.getValueAt(i, 3));
+            final double pIv = (double) pt.getValueAt(i, 5) * 100;
```
Comment (Wolfsblvt) [inline src/me/corriekay/pokegoutil/windows/PokemonTab.java]: I am quite sure the `5` here means the id of the column, right?
First of, we have the policy to don't use magic numbers, but use constants, or at least variables with a explaining name for it.

But here this shouldn't be the fix. columnIds are really not hard-coded. Columns are moveable, etc etc. That's why I've implemented them to be auto-generated.
So to get the ID of the IV column you have to use `PokeColumn.IV_RATING.id`.

Is that understandable?

Or have I missed something in your code? xD

Comment (stevenzeck) [reply src/me/corriekay/pokegoutil/windows/PokemonTab.java]: Yep, I understand. I didn't take a hard look at the newer code, I just knew the selecting wasn't working when casting the getValueAt return object to a String for whatever reason. I will update it to use the enum

Comment (Wolfsblvt) [reply src/me/corriekay/pokegoutil/windows/PokemonTab.java]: Great, thank you. With that, I guess I can merge it :D

Comment (Wolfsblvt) [reply src/me/corriekay/pokegoutil/windows/PokemonTab.java]: No update yet?

**Comment by stevenzeck** [comment]
Sorry, was preoccupied with something.  Committed change using PokeColumn enum to retrieve column index.

**Review by Wolfsblvt — APPROVED** [review]
Yep, working fine now. Thank you.

**Pull request merged**